cluster/addons/kube-proxy/kube-proxy-ds.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         beta.kubernetes.io/kube-proxy-ds-ready: "true"
       initContainers:
       - name: touch-lock
-        image: busybox
+        image: gcr.io/google_containers/busybox:1.24
```

Comment:
please use a newer busybox

Reply:
Ack, will push a newer image and update here.

Reply:
Updated to `gcr.io/google_containers/busybox:1.27`, corresponding to `busybox:1.27`.